src/main/java/org/isf/lab/manager/LabManager.java

```diff
@@ -117,6 +117,19 @@ protected void validateLaboratory(Laboratory laboratory) throws OHDataValidation
 		}
 	}
 
+	/**
+	 * Return the list of exams ({@link Laboratory}s) divided by pages.
+	 * 
+	 * @param oneWeek
+	 * @param pageNo
+	 * @param pageSize
+	 * @return the list of {@link Laboratory}s. It could be <code>empty</code>.
+	 * @throws OHServiceException
+	 */
+	public List<Laboratory> getLaboratory(boolean onWeek, int pageNo, int pageSize) throws OHServiceException {
```

Comment:
The Javadoc has the variable as `oneWeek` (see line 123) and the method signature uses `onWeek`; pick one